jitbuilder/compile/Method.hpp

```diff
@@ -166,7 +167,7 @@ class ResolvedMethod : public ResolvedMethodBase, public Method
                                TR::SymbolReferenceTable *symRefTab);
 
    protected:
-   const char *_fileName;
+   std::string _fileName;
```

Comment:
This change I commented on isn't a JitBuilder "object"; it is not visible to clients of the library. 

Method is a core OMR class that has been extended by the JitBuilder layer. As such, all the memory allocated by its OMR base class(es) will be properly tracked, but if JitBuilder layer is "free" to neglect to use the same Allocator infrastructure, then anything done at the JitBuilder layer will not be similarly tracked.

This path leads to JitBuilder providing misleading answer to the memory tracking features of the OMR compiler / OMR project, because it will only provide part of the answer (with some other part invisibly sitting on the global heap).

Unless I've misunderstood something here, I don't agree with this particular direction.